src/runtime/components/CldVideoPlayer.vue

```diff
@@ -20,6 +20,7 @@ export interface CloudinaryVideoPlayerOptions {
   secure?: boolean;
   transformation?: Array<object> | object;
   hideContextMenu?: boolean;
+  config?: Record<string, any>;
```

Comment:
what do you think about using `ConfigOptions` for this one and the other `config` below?

```
import { ConfigOptions } from '@cloudinary-util/url-loader'
```

Reply:
Added